pkgs/applications/misc/tiramisu/default.nix

```
@@ -1,19 +1,19 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, glib }:
+{ lib, stdenv, fetchFromGitHub, pkg-config, glib, vala }:
 
 stdenv.mkDerivation {
   pname = "tiramisu";
-  version = "unstable-2021-05-20";
+  version = "2.0.20211107";
 
   src = fetchFromGitHub {
     owner = "Sweets";
     repo = "tiramisu";
-    rev = "e53833d0b5b0ae41ceb7dc434d8e25818fe62291";
-    sha256 = "sha256-F4oaTOAQQfOkEXeBVbGH+0CHc9v9Ac08GyzHliOdAfc=";
+    rev = "cdfdec29fd140fa516f2daaad7959c8575ebf91c";
```

Upstream is [tagging](https://github.com/Sweets/tiramisu/releases/tag/2.0.20211107) the source.

```suggestion
    rev = version;
```

---

Fixed. Also set `repo` to `pname` while I was at it.